### PR TITLE
Variational Dropout for Pytorch

### PR DESCRIPTION
**Pull request by blester125**
Adds a Variational Dropout (same mask at each time-step) to Pytorch.

This only works on the input/output of the RNN, not on recurrent connections yet. 

Assumes that input is `(T, B, H)` which is what RNNs already do to to utilize cudnn.

Tagger using Variational Dropout is currently top of tagger conll results in xpctl

### CODE REVIEW

File: python/baseline/tf/classify/model.py [hunk 1]
```diff
@@ -389,9 +389,10 @@ def pool(self, word_embeddings, dsz, init, **kwargs):
         :return: 
         """
         hsz = kwargs.get('rnnsz', kwargs.get('hsz', 100))
+        vdrop = kwargs.get('varitational_dropout', False)
```
Comment (dpressel) [inline python/baseline/tf/classify/model.py]: typo

**Comment by blester125** [comment]
I trained a model on master then checkout this branch and I was able to load and run the model
